### DISCUSSION TITLE
No data

### DISCUSSION THREAD

**Discussion by Danifly80** [klaasnicolaas/home-assistant-glow#261] [Q&A]
Hi!
I'm trying to integrate Glow in my home assistant but I'm struggling with it.

I assembly all the components and I program the ESP32 without a problem, also I integrate the ESP in home assistant, and I can also open the web server without any problem or delay.

The problem is that I can't give any data, the RGB led doesn't flash when the meter flash but It flash blue on startup before connects to wifi.

I check the diode soldering and also I tried to use another diode, with no luck.
I also recheck and use different rgb led.

do you have any Idea?

Esp32 glow regulary update wifi signal on HA

my Meter use a 1Wh/imp value by as you suggest me before I red this[ link](https://github.com/klaasnicolaas/home-assistant-glow/discussions/239) and I didn't modify de default value of "1000"

web server:
> <html><body>

22:31:58 | [D] | [button:013] | 'Reset - Total Energy' Pressed.
-- | -- | -- | --
22:32:02 | [D] | [button:013] | 'Restart - Glow' Pressed.
22:32:02 | [I] | [restart.button:012] | Restarting device...
22:32:02 | [I] | [app:133] | Rebooting safely...
22:32:02 | [D] | [esp32.preferences:114] | Saving preferences to flash...
22:32:47 | [D] | [esp32.preferences:114] | Saving preferences to flash...
22:33:53 | [D] | [sensor:125] | 'House - WiFi Signal': Sending state -68.00000 dBm with 0 decimals of accuracy

</body>
</html>

thanks for the help

**Comment by klaasnicolaas** [comment]
Do you get a response when you shine the flashlight over the photodiode? The PCB of the photodiode connected to the correct GPIO pins?

**Reply by Danifly80** [reply]
Hi
sorry for the waiting but, to be sure I've ordered some new esp32.
unfortunately some result: the led on photodiode flash correctly but It seams like no data are transferred from the photodiode board, no flash on the rgb and no data to home assistant and webserver

I tried with two different photodiode boards but with the same result, both gren led on photodiode flash but no data

thanks for your help

**Reply by klaasnicolaas** [reply]
Good to know that the photodiode board does indicate that a pulse has been detected. Are you using the correct GPIO port?

**Reply by Danifly80** [reply]
I thinks so, I've built two different Glow with two different esp32 board from different vendor and two different photodiode but from the same result.

GPIO port are

ESP32 --> Phtd
3V3 --> VCC
G13 --> DO
GND --> GND

**Reply by Danifly80** [reply]
@klaasnicolaas any other advice?

**Reply by klaasnicolaas** [reply]
But is GPIO13 also set in the yaml code? What version of the yaml code are you using?

**Reply by Danifly80** [reply]
Sorry for my late answer, busy week.

Everything was my fault, linked to my very small experience of esp home (for the moment :D )

I've installed the glow using Crome and https://klaasnicolaas.github.io/home-assistant-glow/

so the version installed was the 3.0 but I connection scheme at this address: https://github.com/klaasnicolaas/home-assistant-glow/tree/v3.1.0

Now I've installed the 3.1.0 and everything work well, I had some problems with some extreme random value but it seams solved with `internal_filter: 100ms`

I will check in the next days if everything is good!

Thanks for the help!
